src/modules/app/containers/__tests__/NavigatorContainer.spec.js

```diff
@@ -40,105 +37,93 @@ class MockNavigator extends React.Component<{}> {
   render () { return null }
 }
 
-// jest.resetModules() produces a invariant violation: Invalid hook call https://github.com/facebook/jest/issues/8987
+let TestRenderer
+let render
+let Provider
+
 describe('NavigatorContainer', () => {
+  beforeEach(() => {
+    jest.resetModules()
+    TestRenderer = require('react-test-renderer')
+    render = require('@testing-library/react-native').render
+    Provider = require('react-redux').Provider
```

Comment:
Why is this required? Maybe add a comment

Reply:
added one